crossplane.hcl

```diff
@@ -0,0 +1,21 @@
+description = "CLI for Crossplane"
+homepage = "https://docs.crossplane.io/latest/cli/"
+binaries = ["crossplane"]
+test = "crossplane --version"
+source = "https://releases.crossplane.io/stable/v${version}/bin/${os}_${arch}/crank"
+
+on "unpack" {
+  rename {
+    from = "${root}/crank"
+    to = "${root}/crossplane"
+  }
+}
+
+version "1.14.5" {}
```

Comment:
You should be able to put an auto-version block in here and have it automatically update:

```hcl
auto-version {
  github-release = "crossplane/crossplane"
}
```